frontend/src/metabase/parameters/components/widgets/ParameterFieldWidget/ParameterFieldWidget.jsx

```diff
@@ -74,6 +76,7 @@ export default function ParameterFieldWidget({
               value={value}
               parameter={parameter}
               parameters={parameters}
+              question={question}
```

Comment:
Woah, what a path it had to take: `PublicQuestion` > `EmbedFrame` > `SyncedParametersList` > `ParametersList` > `ParameterWidget` > `ParameterValueWidget` > `ParameterFieldWidget` > `FieldValuesWidget`

I understand that the answer is probably "no", but is there something we can do to shorten this path?